### PR TITLE
[Bug Fix] IRS Map Indicator Row values

### PR DESCRIPTION
**Pull request by cKellyDesign**
Fix: #482 
Also adds Jurisdiction name to page title

### CODE REVIEW

File: src/containers/pages/IRS/Map/helpers.ts [hunk 1]
```diff
@@ -31,13 +31,16 @@ export const IRSIndicatorStops: { [key: string]: string[][] } = {
...
+export type IndicatorPropIngestor = (obj: any) => string | number;
+
 /** interface to describe and indicator row item */
 export interface IndicatorRowItem {
   denominator: string | number;
   description: string;
   numerator: string | number;
   title: string;
-  value: string | number;
+  value: string | number | IndicatorPropIngestor;
```
Comment (cKellyDesign) [inline src/containers/pages/IRS/Map/helpers.ts]: @moshthepitt how do you feel about this approach of adding custom ingestion methods? Alternatively we can simplify it and just check if `focusArea[row.value] < 1` to handle this use case.

Comment (moshthepitt) [reply src/containers/pages/IRS/Map/helpers.ts]: @cKellyDesign I like this

**Comment by cKellyDesign** [comment]
Oh good point, though I would only expect the snapshot only to fail if the indicator row fixture changed from a key-string to a ingest-method. Will add test.

File: src/containers/pages/IRS/Map/helpers.ts (hunk 1 above)
Comment (cKellyDesign) [reply src/containers/pages/IRS/Map/helpers.ts]: @moshthepitt I actually removed this for the sake of simplicity, i think it's safe to assume that:

1. All progress bar `value`s are represented as percentages
2. Values which are floats need to be multiplied by 100 and rounded
3. Values which are integers should be used as is
4. We never want to render decimals in the indicator row

Do you think these are safe assumptions? If so then we don't need to add the same ingestion method (with different strings) to all the `value` properties.

Comment (moshthepitt) [reply src/containers/pages/IRS/Map/helpers.ts]: @cKellyDesign I think these sound sound :)

**Pull request merged**